Gems/Atom/RPI/Code/Source/RPI.Public/RenderPipeline.cpp

```diff
@@ -301,6 +301,23 @@ namespace AZ
             m_drawFilterMask = 0;
         }
 
+        void RenderPipeline::OnPrepareFrame()
+        {
+            m_lastRenderRequestTime = AZStd::chrono::system_clock::now();
+            if (m_renderMode == RenderMode::RenderAtTargetRate)
+            {
+                constexpr AZStd::chrono::duration<float> updateThresholdMs(0.001f);
+                const bool shouldRender =
+                    m_lastRenderRequestTime - m_lastRenderStartTime + updateThresholdMs >= m_targetRefreshRate;
+                m_rootPass->SetEnabled(shouldRender);
```

Comment:
I feel like this implementation is wasting energy which is precious on mobile plats. You are basically skipping the render and waiting for the next frame hoping that this condition will be true. Instead it may be more efficient to just add a sleep for the exact time you need to throttle this thread for. Thoughts?

Reply:
That's a good callout for mobile, I think we have one of two options here:
1. We can just adjust the tick rate on mobile to be our target refresh rate, in which case this doesn't matter. It's a dial we should probably consider tuning anways?
2. We could also make this logic opt-in by ignoring VSync and only respecting `r_fps_limit` if it's enabled. By default we can still have VSync on and no frame limit, in which case you'll get an immediate draw and wait for vblank. We'll need some other mechanism for figuring out the vblank portion of frame timing to get better frame profiling data in this case (I think that'd be a PAL thing, pretty sure I've seen WinAPI for doing this).

Reply:
I like the idea of Option 1 here. As long it is a sleep and not a spin lock :)

Reply:
I've added `g_simulation_tick_rate` per our conversation, see [this commit](https://github.com/o3de/o3de/pull/3375/commits/0b46290ef4783ebfbe6792eb06c1ea84425d229c) for implementation info.

Testing locally in the Editor/Launcher it's working well on my end and doesn't seem to be interfering with vsync (I get a consistent 60fps when setting `g_simulation_tick_rate` at 60hz with/without vsync, etc)